File: CoreLib_Common/Model/Beaver.cs
namespace CoreLib_Common.Model
{
    //[Table("Beaver")]
    public class Beaver : Animal
    {
        public FluffinessEnum Fluffiness { get; set; }
        public int Size { get; set; }
        public override string ToString()
        {
            return @$"{base.ToString()} Beaver: Fluffiness = {this.Fluffiness} Size = {this.Size}";
        }
    }

    public enum FluffinessEnum
    {
        NotFluffy,
        Fluffy,
        VeryFluffy
    }
}

File: VBLib_Common/Model/Beaver.vb
Namespace Model
    Public Class Beaver
        Inherits Animal

        Public Property Fluffiness As FluffinessEnum
        Public Property Size As Integer

        Public Overrides Function ToString() As String
            Return _
                $"{MyBase.ToString()} Beaver: Fluffiness = {Me.Fluffiness} Size = {Me.Size}"
        End Function
    End Class

    Public Enum FluffinessEnum
        NotFluffy
        Fluffy
        VeryFluffy
    End Enum
End Namespace
